### DISCUSSION TITLE
How can I install TeslaMate on the same raspberry pi 4 I install Tesla Android on?

### DISCUSSION THREAD

**Discussion by sompong-ttbspark** [tesla-android/issue-tracker#278] [Q&A] [answered]
How can I install TeslaMate on the same raspberry pi 4 I install Tesla Android on? Now I'm trying to access my raspberry pi 4 with ssh but I can't access it.

**Comment by mikegapinski** [comment, marked as answer]
No, it's not Linux. There is no ssh, systemctl, apt etc

**Reply by sompong-ttbspark** [reply]
Thank you 👍
